src/config.ts

```diff
@@ -0,0 +1,13 @@
+function getEnvVariable(varName: string): string {
+  const envVar = process.env[varName];
+  if (typeof envVar !== "string") {
+    throw new Error(`Missing '${varName}' env variable`);
+  }
+  return envVar;
+}
+export const SLACK_CRON_PATTERN = getEnvVariable("SLACK_CRON_PATTERN");
+export const SLACK_WEBHOOK_URL = getEnvVariable("SLACK_WEBHOOK_URL");
+export const SLACK_DAY_INTERVAL = parseInt(
```

Comment:
We should throw an error when the value is not an integer. Also, use `Number()` to ensure the whole value is a number.

Reply:
@kiootic is there any reason we can't use `parseInt` directly?